### ISSUE TITLE
"datasources" as domain at title link

### ISSUE THREAD

**Issue by somatorio** [ddurieux/glpi_app_grafana#37]
I just started using this app, so i may forgot to set something, but when i create a table with metrics of tickets i noticed that title come as a `<a></a>` tag (which made me happy, 'cause i could just set "Sanitize HTML" and the link would work), but then i noticed that the "domain" for some reason was appearing as "datasources" just like below
![image](https://user-images.githubusercontent.com/7087238/48019833-89f98d80-e11b-11e8-8fab-2a6eb6c506bf.png)

Is it a bug/issue or i just set something wrong?

**Comment by ddurieux** [comment]
The `Titulo` has `Satinize HTML`?

**Comment by somatorio** [comment]
To take the screenshot i unchecked `Sanitize HTML` to show the tag content :)

**Comment by ddurieux** [comment]
Yes it's normal, but when you check `Sanitize HTML` it show the name normally no?

**Comment by somatorio** [comment]
Yeah, but my issue isn't the sanitize HTML part, check the screenshot at the href part... The link points to "http://datasources/" and this domain (obviously) isn't where my glpi is hosted at

**Comment by ddurieux** [comment]
Ah yes, sorry , I'm tired :p
Is your GLPI URL right defined in GLPI (menu setup > general) ?

**Comment by somatorio** [comment]
Yes it is :)

**Comment by romulf** [comment]
hello Somatorio,
did you find a solution? I'm facing the same issue...
Regards,

**Comment by somatorio** [comment]
Nope :(

**Comment by FabienZouaoui** [comment]
Hi there,

As a workaround, you can set Access to "Browser" instead of "Server (default)" in datasource config to solve this.
